test/test_config.py

```diff
@@ -71,3 +73,12 @@ def test_header_encoding_is_reflected(self, header_encoding):
         config = h2.config.H2Configuration()
         config.header_encoding = header_encoding
         assert config.header_encoding == header_encoding
+
+    def test_logger_instance_is_reflected(self):
+        """
+        The value of ``logger``, when set, is reflected in the value.
+        """
+        logger = logging.Logger('hyper-h2.test')
+        config = h2.config.H2Configuration()
+        config.logger = logger
```

Comment:
We don’t have any other tests that exercise the `H2Configuration` constructor – I’ve opened #454 to track fixing that.